sample-lnd.conf

```diff
@@ -489,10 +489,12 @@ bitcoin.simnet=true
 ; Use the btcd back-end
 bitcoin.node=btcd
 
-; Use the bitcoind back-end
+; Use the bitcoind back-end.
 ; bitcoin.node=bitcoind
 
 ; Use the neutrino (light client) back-end
+; WARNING: Please note switching from a full back-end (btcd/bitcoind) to 
```

Comment:
nit: add full stop in line above as well?
Also: please note _that_ switching...